scripts/hudson/narayana-rebase.sh

```diff
@@ -56,7 +56,7 @@ function rebase_narayana {
   fi
 
   # Update the pull to head  
-  git pull --rebase --ff-only origin $BRANCHPOINT
+  git pull --rebase origin $BRANCHPOINT
```

Comment:
Thank you Manuel for your review! 
> About "In fact now the rebase always fails if there is a new commit on top of the main branch.". Why a contributor should add a commit on top of the main branch when raising a PR?
I will describe a use-case here:
- there are 2 PRs for Narayana main. Both are tested and tests passed. A reviewer asked for a change for the second PR, the change is addressed and changes are pushed. The first PR is merged. Tests start again for the second PR to test the changes are ok. The test will fail because in the meanwhile the first PR has been merged to main and the HEAD main changed. 
This is even more complicated because tests do not start automatically and so we cannot foretell when they will start and weather other PRs will be merged in the meanwhile.
However I understand this might happen only sometimes, if this is causing more troubles than benefits I am OK with closing this PR.